src/cf.data.pre

```diff
@@ -10301,29 +10301,41 @@ DOC_START
 	reduced memory thrashing in your malloc library.
 DOC_END
 
-NAME: forwarded_for
+NAME: http_header_forwarded forwarded_for
```

Comment:
I am not yet convinced that we need to rename this directive. Why is renaming better than keeping the current name? Please share your cost/benefit analysis.

FWIW, if we do end up renaming, I am pretty sure that http_header_forwarded is not a good name. Something like forward_client_addresses would be a lot better IMO.